aries-site/src/pages/templates/lists.mdx

```diff
@@ -1,16 +1,109 @@
 import { Example } from '../../layouts';
-import { ListScreenExample } from '../../examples';
+import {
+  ListIconIdentifierExample,
+  ListImageIdentifierExample,
+  ListNameOptionActionExample,
+  ListNameActionExample,
+  ListNameDescriptionOptionExample,
+  ListScreenExample,
+} from '../../examples';
 
 HPE Design System list view templates are go-to patterns for displaying homogeneous data such as services, devices, users, and more.
-
-List views are optimized for scanability and reading comprehension. Each list item provides users focussed information and identity labels to aid selection, decision making, and action.
-
-## Manage Users Screen
+List views are optimized for scanability and reading comprehension. Each list item provides users focused information and identity labels to aid selection, decision making, and action.
 
 <Example
   code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/list-views/ListScreenExample.js"
+  docs="https://v2.grommet.io/list?theme=hpe#props"
   template
   width="100%"
 >
   <ListScreenExample />
 </Example>
+
+## Guidance
+
+Lists are an effective way to display dense content to users. Here are some recommendations on how to make the data easily consumable.
+
+**Best Practices:**
+
+- Be consistent with layout and visual treatments for all list items.
+- Keep list item descriptions brief and concise.
+- Use Icons or Images in list items to help draw attention and support scanning.
+- Maintain sufficient space between list items or consider using dividers between rows.
+- Sort lists in a logical order such as alphabetically, numerically, chronologically, or by user preference.
+
+**Lists Types:**
+
+- Single-line lists are optimal for scan-ability, especially with the addition of an image.
+- Two-line lists provide a primary and secondary text which can be helpful when more detail is needed.
+- Three-line lists of a primary text and two lines of secondary text are not recommended.
+- Grid lists are made of image tiles and require more space.
+
+## Variants
+
+Below are some common list variations that demonstrate how to combine visual content, text, and list actions within the component.
+
+### Image + Name + Description + Action
+
+Two-line list with an image, primary title, description, and an action button.
```

Comment:
In each of these variants, I feel like we should be elaborating on the unique element being added to the example. In this case, the image is what is being added, so we should explain how it should be used to assist scan-ability -- specific to this example, the image should be tied to either the User or the Location. If for the User, each image would be unique and assist the end-user in making a deeper cognitive connection with each User in the list; if for the location, then allows the end-user to infer common associations / mental groupings between individual users.

Reply:
Basically, this relates to discussion about "we are doing a good job with the 'What', but need to get better in our guidance about filling in the 'Why', 'How', and 'When' to use.

Reply:
I agree completely. I put this in as starter text, but was thinking Cheri would be able to add in more details

Reply:
Just put a summary of my thoughts in a review comment. Code-wise things look good. Main reactions are to the content and story we are telling. Don't know the best way to proceed. Perhaps, do like Page Layouts --> publish --> demo and/or summarize with callouts for areas needing attention / thought starters -->  get folks to react and contribute their thoughts ???

Reply:
Agree with Matt's comment about ordering the examples from simple to complex, allowing you to build on the previous ones.

Would like to see examples of other screen sizes too, especially mobile.

Feel free to add a github issue for me to help out with this, with what is most needed.

Reply:
@cherih I've created an issue here summarizing some of the content additions we'd like to add to this page: https://github.com/hpe-design/design-system/issues/964

Reply:
@taysea - thanks I'll take a look at #964.

Reply:
Just thinking about the Figma design examples some more and the previous comments about ordering examples.

Is it possible to start with the simplest example of all (i.e. Name only)? 
Then create the more complex examples step by step.

Also, maybe include fewer examples with the Actions menu in the list?
We are advising against using it for GLC. It was confusing to users in usability tests and it doesn't support multi-select.